api/types/lock.go

```diff
@@ -201,28 +201,14 @@ func (t LockTarget) Match(lock Lock) bool {
 		return false
 	}
 	lockTarget := lock.Target()
-	if t.User != "" && lockTarget.User != t.User {
-		return false
-	}
-	if t.Role != "" && lockTarget.Role != t.Role {
-		return false
-	}
-	if t.Login != "" && lockTarget.Login != t.Login {
-		return false
-	}
-	if t.Node != "" && lockTarget.Node != t.Node {
-		return false
-	}
-	if t.MFADevice != "" && lockTarget.MFADevice != t.MFADevice {
-		return false
-	}
-	if t.WindowsDesktop != "" && lockTarget.WindowsDesktop != t.WindowsDesktop {
-		return false
-	}
-	if t.AccessRequest != "" && lockTarget.AccessRequest != t.AccessRequest {
-		return false
-	}
-	return true
+	return (t.User == "" || lockTarget.User == t.User) &&
+		(t.Role == "" || lockTarget.Role == t.Role) &&
+		(t.Login == "" || lockTarget.Login == t.Login) &&
+		(t.Node == "" || lockTarget.Node == t.Node) &&
+		(t.MFADevice == "" || lockTarget.MFADevice == t.MFADevice) &&
+		(t.WindowsDesktop == "" || lockTarget.WindowsDesktop == t.WindowsDesktop) &&
+		(t.AccessRequest == "" || lockTarget.AccessRequest == t.AccessRequest) &&
+		(t.Device == "" || lockTarget.Device == t.Device)
```

Comment:
i am not sure if this is more readable but not strong opinions here.

Reply:
👍